app/views/games/show.html.erb

```diff
@@ -37,9 +42,11 @@
             document.getElementById("q").src = q.src.replace("white", "black");
             document.getElementById("k").src = k.src.replace("white", "black");
             document.getElementById("r").src = r.src.replace("white", "black");
-            document.getElementById("b").src = b.src.replace("white", "black");                        
+            document.getElementById("b").src = b.src.replace("white", "black");      
             $('#myModal').modal('show');
           }
+          else if (x === horiz || y === vert) {
```

Comment:
I think maybe this should have && instead or || Right now it's preventing a window reload for all non-diagonal moves